README.md

```diff
@@ -1,6 +1,46 @@
 # Bike Sharing Sample Application
 
-## Docker-Compose usage
+## Azure Dev Spaces
+
+1. Create an AKS cluster with Azure Dev Spaces by running the following script, specifying the AKS name and region:
+    
+    ```
+    source ./create-aks.sh bikesharing01 eastus2
+    ```
+
+    This sets up an AKS cluster named `bikesharing01` in resource group `bikesharing01` in the `eastus2` region.
+
+2. Set the environment variable for Gateway API Service URL.
+
+    a) Check your AKS cluster's zone name: 
+    
+    `az aks show -g $AKS_NAME -n $AKS_NAME -o json --query addonProfiles.httpApplicationRouting.config.HTTPApplicationRoutingZoneName`
+
+    b) Open the web frontend's azds.yaml: `code ./bikesharingweb/azds.yaml`
```

Comment:
I thought we wanted to guide users to update this in the web frontend's values.yaml rather than the azds.yaml?  Since its not necessarily an AZDS-specific setting.